h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -153,9 +176,6 @@ private void parseArgs(String[] args) {
             nPlaces = Integer.parseInt(s);
             break;
 
-          case "--raw":
```

Comment:
should be backward compatible, please keep this option - it can be removed from help but it should keep working